internal/lib/sandbox/namespaces_linux.go

```diff
@@ -176,29 +174,14 @@ func (n *Namespace) Remove() error {
 
 	n.closed = true
 
-	// we got namespaces in the form of
-	// /var/run/$NSTYPEns/$NSTYPE-d08effa-06eb-a963-f51a-e2b0eceffc5d
-	// but /var/run on most system is symlinked to /run so we first resolve
-	// the symlink and then try and see if it's mounted
-	fp, err := securejoin.SecureJoin("/", n.Path())
-	if err != nil {
-		return errors.Wrapf(err, "unable to join '/' with %s path", n.Path())
-	}
```

Comment:
Yeah this is all quite :thinking:-worthy. And :+1: on removing the needless "is it mounted" check.